File: Coders/CursoRegex/conjuntos/intervalo.py
import re

texto = '1,2,3,4,5,6,a.b c!d?e[f'

print(re.findall(r'[1-3]', texto))
print(re.findall(r'[b-e]', texto))
print(re.findall(r'[A-Z1-3]', texto, re.IGNORECASE))

File: coders/curso_regex/conjuntos/intervalo.js
const texto = '1,2,3,4,5,6,a.b c!d?e[f'

console.log(texto.match(/[1-5]/g))
console.log(texto.match(/[b-e]/g))
console.log(texto.match(/[A-Z1-3]/gi))
